src/PresentationalComponents/ExecuteTaskButton/ExecuteTaskButton.js

```diff
@@ -61,7 +61,7 @@ ExecuteTaskButton.propTypes = {
   setIsRunTaskAgain: propTypes.func,
   setModalOpened: propTypes.func,
   slug: propTypes.string,
-  title: propTypes.string,
+  title: propTypes.any,
```

Comment:
I think that the title might be either string or a React element. If this is the case, can we use more explicit prop type. 

```suggestion
  title: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.elementType,
  ])
```